tests/testthat/test-multitables.R

```diff
@@ -232,6 +268,23 @@ with_test_authentication({
                                               "  selected_array(allpets)",
                                               "  q1"))
     })
+    
+    test_that("Can make a multitable with list methods", {
+        multitables(ds)[["new mt"]] <- ~ds$country+ds$q3
```

Comment:
This could also be specified as `as.formula(~country+q3, ds)`. If we want the user to be able to specify the formula without the dataset reference, I'll need add in code that tracks down what the dataset that the multitable belongs to. This is probably the better solution because it lets us check that the variables the user is specifying actually exist in the dataset.